helm/kyverno-policies-connectivity/templates/Pod.yaml

```diff
@@ -16,6 +16,11 @@ spec:
             names:
             - "capi-*"
             - "dex-k8s-*"
+        - resources:
+            kinds:
+            - Pod
+            annotations:
+              policies.kyverno.io/inject-proxy-env: block
```

Comment:
I'd suggest using `disable` or `exclude`, `block` sounds strange to me, but it may be a personal thing so feel free to ignore this if you prefer `block`.

Reply:
using exclude as its same syntax as in kyverno